httpclient/src/main/kotlin/tbdex/sdk/httpclient/TbdexHttpClient.kt

```diff
@@ -17,12 +17,13 @@ import tbdex.sdk.protocol.models.Offering
 import tbdex.sdk.protocol.serialization.Json
 import tbdex.sdk.protocol.serialization.Json.jsonMapper
 import web5.sdk.dids.Did
+import java.time.Duration
 
 /**
  * A client of the tbDEX HTTP interface for communicating with a PFI.
  */
 object TbdexHttpClient {
-  private val client = OkHttpClient()
+  private var client = OkHttpClient()
```

Comment:
So, we originally made `TbdexHttpClient` an object because we didn't think people needed to instantiate it/one would be enough. However, if we're making the timeout customizable, I think we could make it a normal class, and leave `OkHttpClient` as val. So effectively you could pass in the timeout as an argument to `TbdexHttpClient`'s constructor, which would set the timeout on OkHttpClient. 

The reason for doing this is that it isn't immediately clear when you call `setTimeout` that you are constructing a new `OkHttpClient` every time. (Builder.build returns a new instance of the client).

Reply:
Check out this section about `init` blocks for how you can use arguments from the constructor when creating a new instance of a class: https://kotlinlang.org/docs/classes.html#constructors